husky_gazebo/launch/controller_utils.launch

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
```

Comment:
Move over to husky_control, and we can figure out a way to merge with the existing stuff, probably using arg passing.